### DISCUSSION TITLE
Eğitime destek vermek tarihi ve kültürel yerlerin tanıtım ve etkinlikleri

### DISCUSSION THREAD

**Discussion by Sadikizal** [git-lfs/git-lfs#5252] [Ideas] [answered]
Eğitime destek vermek etkinliklere her etkinlikte yeni bir uygulama ile yeni bir tarihi kültürel ögelerin tanıtımıyla katkı sağlamak yorumlara açıktır

**Comment by bk2204** [comment, marked as answer]
This doesn't seem to be a helpful discussion, so I'm going to lock it.  Translating this from Turkish to English doesn't indicate that this is related to Git LFS or even programming in general.

We're effectively limited here to helping in English, Spanish, French, and German, since those are the languages that the core team knows.
